README.md

```diff
@@ -73,7 +73,7 @@ and we need your help to make Matomo better… Why not participate in a useful p
 
 The Matomo project uses an ever-expanding comprehensive set of thousands of unit tests and hundreds of automated integration tests, system tests, JavaScript tests, and screenshot UI tests, running on a continuous integration server as part of its software quality assurance. [Learn more](https://developer.matomo.org/guides/tests)
 
-We use [BrowserStack.com](https://www.browserstack.com/) testing tool to help check the Matomo user interface is compatible with many browsers.
+We use [BrowserStack.com](https://www.browserstack.com/) testing tool to help check the Matomo user interface that is compatible with many browsers.
```

Comment:
The sentence doesn't make any sence with the added `that`. If one insists on adding it, then it should be after check.

Reply:
Hi, I am just starting out with contributing to open source, I will try to give much better contributions frim now on, it will be good if you can help me with this.